src/tests/ftest/dfuse/daos_build.yaml

```diff
@@ -3,8 +3,8 @@ hosts:
     - server-A
   test_clients:
     - client-C
-# Test typically takes around 15 minutes, set timeout for 30.
-timeout: 1800
+# Hard limit the test at four hours, however individual tests have lower timeouts.
+timeout: 14400
```

Comment:
FYI that this timeout applies to EACH mux variant. So in total the timeout is 14400 * 4 variants

Reply:
8.  It's really just a good idea to have a number in here, it's a test error if we get to this far anyway.

Reply:
It's even **much** better to have an **accurate** number in here as this is how long a test that spins and makes no progress can tie up a cluster.  In this case it would be 16 hours.  And on the other hand, if the tests really do take (near) 16 hours, I cannot possibly see how we could do this daily.

Reply:
They don't.  Actual timeouts are in the test and are much lower.  This is just an escape net.

Reply:
But if it is an escape net, it should be accurate.  If nothing else it should reflect, to someone looking through the yaml files, how long a test is expected to take.  Or how long we are willing to let it take.  Somebody could be using that value, for example to decide if a test can fit into a stage or test run.  So in this case, if the test really only takes 20m (to just use a random number to make a point) but you are giving it 4h, somebody could see 4h and decide that that is too long for a given test run.

But more importantly, if the escape net really is needed, let's make it a useful escape net?

Reply:
OK.  Fair enough.  Just so long as we don't forget to zone it in once more testing has been done.